src/elements/content-sidebar/versions/messages.js

```diff
@@ -187,6 +187,17 @@ const messages = defineMessages({
         defaultMessage: 'Retention policy expires on {time}.',
         description: 'Message describing when the version retention policy will expire.',
     },
+    versionUpsell: {
+        defaultMessage:
+            'Sorry, version history is not available with your current account plan. To access versioning, select from one of our paid plans.',
+        description: 'upgrade now description',
```

Comment:
might be helpful for our translators to add a little more information to the message description